### ISSUE TITLE
Position After PowerView Automation

### ISSUE THREAD

**Issue by davidvanvo** [GvnCampbell/homebridge-powerview#1]
I still have my PowerView Automation configured to close and shut based on sunset and sunrise. Noticed that when this is activated, Homekit shows 50% Open even though it's fully closed or fully open.

**Comment by GvnCampbell** [comment]
Yup.  That is a known issue. Sometimes the hub doesn't report back where the blinds are as it loses track of them.  So what I did was add a default option.  This way you can set it to something like 49% if it doesn't know where it is and when the automation kicks in to set it to 0 or 100 it will still move it there.

I think I may have found a way to solve this by sending a jog command to the blinds when they don't return the location variable, but I still have to test this out.  For now though you will see it set wrong every now and then when they are out of sync.  Once it adjusts it, it will figure out where it is again.

**Comment by Jamelele** [comment]
It seems that when you check the position of the blind in the app, it updates the position in the API. So there must be a way...I’ll investigate the traffic from the app.

**Comment by GvnCampbell** [comment]
@Jamelele Been looking for a solution to that.  It bugs me too, but I have not been able to find any sort of call that would update the position automatically.  I found a Jog command but I don't want to use this as it would put extra strain on the battery and randomly job the blinds.  If you find anything with the traffic let me know.  I'm not that good at tracing network traffic.
